sdcm/cluster.py

```diff
@@ -2214,6 +2214,7 @@ def install_scylla(self, scylla_repo):
         self.remoter.run('curl -s "https://get.sdkman.io" | bash')
         self.remoter.run(shell_script_cmd("""
             source "$HOME/.sdkman/bin/sdkman-init.sh"
```

Comment:
you need to run this source command, again after the install...

Reply:
nope, actually the issue is that, we are running somewhere `sudo nodetool flush` and sudo runs with a different user, that is still looking at the wrong `jdk` version
i need to update that, to have it set to all, not only test user

Reply:
I've raised an SCT issue for the usage of root in the upgrade tests:
https://github.com/scylladb/scylla-cluster-tests/issues/4809